Firestore/Source/Local/FSTWriteGroup.mm

```diff
@@ -34,6 +33,7 @@
 
 namespace Firestore {
 
+// TODO(gsoltis): consider porting something like this to leveldb_transaction.
```

Comment:
We only construct the descriptions when debugging so you could factor out the stuff that constructs a `WriteBatch` from `Commit()` and just reuse this directly.

Reply:
As discussed, I will do this once the PR to convert `descriptionForKey` to C++ lands.